### ISSUE TITLE
False positive

### ISSUE THREAD

**Issue by sindresorhus** [dustinspecker/eslint-plugin-no-use-extend-native#26]
I got a false positive with this code:

```js
var stack = new Error().stack.slice(1);
```

More specifically, the `.slice()` call.

`.stack` is just a string, so `.slice()` shouldn't cause a violation.

**Comment by dustinspecker** [comment]
Thank you for the report. I'll publish a patch in the next few minutes.
